src/kv/test/kv_bench.cpp

```diff
@@ -14,12 +14,11 @@ using namespace ccfapp;
 using namespace ccf;
 
 // Helper functions to use a dummy encryption key
-ccf::LedgerSecrets create_ledger_secrets()
+std::shared_ptr<ccf::LedgerSecrets> create_ledger_secrets()
```

Comment:
Was does this need to be a shared pointer?

Reply:
The `TxEncryptor` now takes a shared pointer to the ledger secrets as it needs it during compaction for sealing, etc.

Reply:
can it not have a reference to the object?

Reply:
As discussed, shared pointer is OK here.